android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/EditTextFactory.java

```diff
@@ -274,4 +282,25 @@ private void addNumericIntegerValidator(JSONObject jsonObject, MaterialEditText
             }
         }
     }
+
+    private void addRelativeNumericIntegerValidator(JSONObject editTextJSONObject, JsonFormFragment formFragment, MaterialEditText editText) throws JSONException {
+        JSONObject relativeMaxValidationJSONObject = editTextJSONObject.optJSONObject(V_RELATIVE_MAX);
+        if (relativeMaxValidationJSONObject != null) {
+            // validate that the relative max field exists
+            String relativeMaxValidationKey = relativeMaxValidationJSONObject.optString(JsonFormConstants.VALUE);
+            JSONObject formJSONObject = new JSONObject(formFragment.getCurrentJsonState());
+            JSONArray formFields = fields(formJSONObject, STEP1);
+            JSONObject relativeMaxFieldJSONObject = getFieldJSONObject(formFields, relativeMaxValidationKey);
+            if (relativeMaxFieldJSONObject != null) {
+                String relativeMaxValidationErrorMsg = relativeMaxValidationJSONObject.optString(JsonFormConstants.ERR);
+                String defaultErrMsg = String.format(DEFAULT_RELATIVE_MAX_VALIDATION_ERR, relativeMaxValidationKey);
+                // add validator
+                editText.addValidator(new RelativeMaxNumericValidator(
+                        relativeMaxValidationErrorMsg == null ? defaultErrMsg : relativeMaxValidationErrorMsg,
```

Comment:
if the err message was not specified then relativeMaxValidationErrorMsg will be empty string, if you want to check using null the use   
`String relativeMaxValidationErrorMsg = relativeMaxValidationJSONObject.optString(JsonFormConstants.ERR,null);`

Reply:
Done.